### ISSUE TITLE
Disabling source map warnings?

### ISSUE THREAD

**Issue by phyllisstein** [postcss/postcss-loader#241]
Hey there! I hate to be a nudzh about a package that's so invaluable, but the warnings about source maps introduced in recent versions are a little overwhelming:

<img width="489" alt="screen shot" src="https://cloud.githubusercontent.com/assets/2207980/26298457/5da21b7c-3ea5-11e7-810e-cbba4caffa5e.png">

I deliberately let source maps get dropped on the floor because they break hot reloading for me and I haven't had the time to figure out why; so every time my bundle reloads I get a huge number of these---13 is a relatively small number in a relatively small project---and they're tending to drown out other warnings and errors from Webpack and React.

Although I can understand the impulse to be helpful to new users, would it be callous to suggest that this is kind of an RTFM situation and emitting a slew of warnings for every build is overkill?

**Comment by michael-ciniawsky** [comment]
@phyllisstein Just remove `{ sourceMap: true }` from all preceding loaders and the warning should be gone (e.g `sass-loader`), there is sadly no way to display warnings like that only once and therefore won't fix from an implementation point of view sry

**Comment by phyllisstein** [comment]
Understood! Thanks for the quick response. Looks like `resolve-url-loader` was doing something funky and always emitting source maps, which was the _actual_ proximate cause of the frustration. Sorry for the bother. 😬

**Comment by michael-ciniawsky** [comment]
`resolve-url-loader` requires `sourceMap`  to work correctly, but that's another issue 😄 :stuck_out_tongue:
